### ISSUE TITLE
Add package.json and README to npm package

### ISSUE THREAD

**Issue by JoshuaGross** [orlandohill/waxeye#18]
Not sure where the npm package files are (src/javascript?) or who's maintaining it (@leiko?) but it would be very helpful to have a package.json for versioning and a README in the NPM project.

**Comment by JoshuaGross** [comment]
Hm, I appear to be mistaken. There is a package.json. Not sure how I missed that. Sorry!

**Comment by JoshuaGross** [comment]
So, for auditing purposes within Yahoo, it would be nice to have a repo somewhere that contains the package.json (a README would be bonus, but not necessary). Any chance of that happening?

**Comment by orlandohill** [comment]
Yes, I hope to restart active development on Waxeye in about a month. I haven't talked with Maxime yet, but I imagine I'd take over maintenance of what gets published to npm and add the required files to the main repo.

**Comment by maxleiko** [comment]
Hi there. No problem for the ownership of the npm package. I just did it a while ago because I'm actively using it in Kevoree :)

**Comment by maxleiko** [comment]
And I'd rather have it as an npm dependency instead of a raw source-file in my project
